_i18n/es.yml

```diff
@@ -1206,7 +1206,7 @@ accessibility:
 
     En el siguiente enlace se pueden consultar nuestros informes de evaluación de accesibilidad realizados:
 
-    [Informe de creación de Cuenta IAL 1](https://docs.google.com/document/d/1wqbVOvNg-7tFW1l4ZxMJrA_VR2gQqNpimlbzxPtvpXg/export?format=pdf)
+    [Informe de creación de Cuenta IAL 1](site.baseurl/docs/accessibility-assessment-ial1-account-creation.pdf)
```

Comment:
so this does turn into `/es/docs/accessibility....pdf`... but it turns out, that's fine, the PDF is downloadable from that URL too